client/pages/_app.js

```diff
@@ -27,7 +27,6 @@ function MyApp({ Component, pageProps }) {
       </Head>
       <Header />
       <Container
-        overflow="hidden"
```

Comment:
This is hindering the select display for menu items in `recycle-and-reuse` page because the menu gets clipped off. @CippyLOL , you were the one who added this constraint for overflow. Do you remember why? I would like to remove it so that the select display works properly, but I'm not sure if this will break anything...